### ISSUE TITLE
Add function for print label

### ISSUE THREAD

**Issue by Moosjes** [apenab/react-dymo#1]
Hello,

I like your work at react-dymo-hooks! It is very easy to read status and printers. 

I do not see how I can print using one of the printers. Could you tell me how to do that?

Thank you!

**Comment by apenab** [comment]
Here an example.

```javascript
import {dymoRequestBuilder} from "react-dymo-hooks";

const params = {
        data: `printerName=${encodeURIComponent(printerSelected)}&printParamsXml=&labelXml=${encodeURIComponent(
            xml
        )}&labelSetXml=`,
};

function handlePrintLabel() {
        dymoRequestBuilder({
            method: "POST",
            wsAction: "printLabel",
            axiosOtherParams: params,
        })
         .then(() => {})
         .catch(() => {});
}
```

**Comment by apenab** [comment]
I think that is a good point create a utility funcion for this issue.

**Comment by Moosjes** [comment]
Thanks!

**Comment by nicolasmeijer** [comment]
Ok just saw this issue. Could be good to put it in the docs as well.

**Comment by apenab** [comment]
> Ok just saw this issue. Could be good to put it in the docs as well. 

I going to add this explanation in the Readme.md doc today

**Comment by sarkis1997** [comment]
Great job on the react-dymo- hooks! In the web sdk of dymo there is an option to print multiple labels in one job. This could be achieved by the LabelSetBuilder. How can I achieve the same, to build multiple labels in one job?

**Comment by apenab** [comment]
> Great job on the react-dymo- hooks! In the web sdk of dymo there is an option to print multiple labels in one job. This could be achieved by the LabelSetBuilder. How can I achieve the same, to build multiple labels in one job?

I'll create an issue for this task and I'll analyze the possibility to achieve this task(build multiple labels in one job)

**Comment by apenab** [comment]
> Great job on the react-dymo- hooks! In the web sdk of dymo there is an option to print multiple labels in one job. This could be achieved by the LabelSetBuilder. How can I achieve the same, to build multiple labels in one job?

Discussion about this topic here: https://github.com/apenab/react-dymo/discussions/33

**Comment by sarkis1997** [comment]
Hi,I saw that you have implemented this function.Do you have an example of the string to pass for multiple prints?Regards,SarkisOp 23 okt. 2022 om 16:51 heeft Antonio Peña Batista ***@***.***> het volgende geschreven:﻿
Closed #1 as completed via #35.

**Comment by apenab** [comment]
> Hi,I saw that you have implemented this function.Do you have an example of the string to pass for multiple prints?Regards,SarkisOp 23 okt. 2022 om 16:51 heeft Antonio Peña Batista ***@***.***> het volgende geschreven:﻿ Closed #1 as completed via #35. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

@sarkis1997 https://github.com/apenab/react-dymo/issues/34#issuecomment-1288156320
